### PR TITLE
Add slash to network storage output for pre-existing file systems

### PR DESCRIPTION
**Pull request by heyealex**
The slash was added for the mount scripts directly, but not in the network storage output directly.

### Submission Checklist

* [x] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [x] Are all tests passing? (`make tests`)
* [x] Have you written unit tests to cover this change?
* [x] Is unit test coverage still above 80%?
* [x] Have you updated all applicable documentation?
* [x] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

File: modules/file-system/pre-existing-network-storage/outputs.tf [hunk 1]
```diff
@@ -18,7 +18,7 @@ output "network_storage" {
   description = "Describes a remote network storage to be mounted by fs-tab."
   value = {
     server_ip             = var.server_ip
-    remote_mount          = var.remote_mount
+    remote_mount          = contains(local.mount_vanilla_supported_fstype, var.fs_type) ? local.remote_mount_with_slash : var.remote_mount
```
Comment (nick-stroud) [inline modules/file-system/pre-existing-network-storage/outputs.tf:21]: nit: from a readability standpoint it is getting a bet hard to follow when we need remote_mount_with_slash and when we just use remote mount. I feel like there is maybe an opportunity to say early on, "is it a gcs bucket, if not add the slash, and then just use remote mount for the rest of the module". If you think it could be easily implemented go for it otherwise feel free to ignore.

**Review by nick-stroud — APPROVED** [review]

File: modules/file-system/pre-existing-network-storage/outputs.tf (hunk 1 above)
Comment (heyealex) [reply modules/file-system/pre-existing-network-storage/outputs.tf]: I agree, and made the change. I'm running a couple tests before actually merging just to be safe.
